feat(replay): Add docs for canvas support in Replay (#8879)
getsentry/sentry-docs: src/platforms/javascript/common/session-replay/troubleshooting.mdx

This guide aims to extend the <PlatformLink to="/troubleshooting/">main troubleshooting</PlatformLink> guide by covering Replay-specific scenarios.

## My `canvas` elements aren't getting captured

> Canvas is supported in SDK versions >= `7.98.0`. Please see the <PlatformLink to="/session-replay/#canvas-recording">canvas setup documention</PlatformLink> for getting started with canvas recordings.

Author: ```suggestion
Canvas is supported in SDK versions >= `7.98.0`. Please see the <PlatformLink to="/session-replay/#canvas-recording">canvas setup documention</PlatformLink> to get started with canvas recordings.
```

> ## My `canvas` Elements Aren't Getting Captured

Reviewer: ```suggestion
## My `canvas` elements aren't getting captured
```

> Canvas is supported in SDK versions >= `7.94.0`. Please see the <PlatformLink to="/session-replay/#canvas-recording">canvas setup documention</PlatformLink> for getting started with canvas recordings.

Author: TBD on version (current is 7.93.0).

> Canvas is supported in SDK versions >= `7.94.1`. Please see the <PlatformLink to="/session-replay/#canvas-recording">canvas setup documention</PlatformLink> for getting started with canvas recordings.
> 

Reviewer: ```suggestion
Canvas is supported in SDK versions >= `7.94.1`. Please see the <PlatformLink to="/session-replay/#canvas-recording">canvas setup documention</PlatformLink> to get started with canvas recordings.

```

## Replay is slowing down my `canvas`

> In order to export images from 3d and WebGL canvases, the integration needs to enable `preserveDrawingBuffer` which can negatively affect canvas performance. If your canvas application is impacted by enabling `preserveDrawingBuffer`, then you will need to <PlatformLink to="/session-replay/#canvas-recording-performance">enable manual snapshotting</PlatformLink> and call a `snapshot()` method inside of your re-paint loop.

Reviewer 2: ```suggestion
In order to export images from 3D and WebGL canvases, the integration needs to enable `preserveDrawingBuffer` which can negatively affect canvas performance. If your canvas application is impacted by enabling `preserveDrawingBuffer`, then you will need to <PlatformLink to="/session-replay/#canvas-recording-performance">enable manual snapshotting</PlatformLink> and call a `snapshot()` method inside of your re-paint loop.
```

> In order to export images from 3D and WebGL canvases, the integration needs to enable `preserveDrawingBuffer` which can negatively affect canvas performance. If your canvas application is impacted by enabling `preserveDrawingBuffer`, then you will need to <PlatformLink to="/session-replay/#canvas-recording-performance">enable manual snapshotting</PlatformLink> and call a `snapshot()` method inside of your re-paint loop.

Reviewer: ```suggestion
The integration needs to enable `preserveDrawingBuffer` to export images from 3D and WebGL canvases. This can negatively affect canvas performance. If your canvas application is impacted by enabling `preserveDrawingBuffer`, you'll need to <PlatformLink to="/session-replay/#canvas-recording-performance">enable manual snapshotting</PlatformLink> and call a `snapshot()` method inside of your re-paint loop.
```

> ## My Custom CSS/Images/Fonts/Media Aren't Appearing When I View the Replay

Reviewer: ```suggestion
## My custom CSS/Images/Fonts/Media aren't appearing when I view the Replay
```
